test_python/test_triangulateContours.py

```diff
@@ -0,0 +1,24 @@
+from helper import *
+import pytest
+
+def test_triangulateContours():
```

Comment:
according to [PEP8](https://peps.python.org/pep-0008/)

```
PEP 8: E302 expected 2 blank lines, found 1
Function name should be lowercase
```

```suggestion

def test_triangulate_contours():
```